Cargo.toml

```diff
@@ -15,6 +15,12 @@ documentation = "https://docs.rs/simple_accumulator"
 
 [dependencies]
 num = "0.4.0"
+rand = "0.8.5"
+plotly = "0.6.0"
```

Comment:
Make `plotly` a dev-dependency for the library. Is someone just need the library, as shepherd is going to, it must not pull in plotly.

Reply:
See https://stackoverflow.com/questions/35711044/how-can-i-specify-binary-only-dependencies @purnatag

Reply:
This is done.